openedx/core/djangoapps/embargo/tests/test_api.py

```diff
@@ -175,10 +175,10 @@ def test_caching(self):
             # (restricted course, but pass all the checks)
             # This is the worst case, so it will hit all of the
             # caching code.
-            with self.assertNumQueries(3):
+            with self.assertNumQueries(5):
```

Comment:
This is because of how `any` works right? in that all are evaluated? because you would expect with how you used logical or operators, which should short cut after the first check (which is almost always the legacy check)? Or is there another reason this changed?

Reply:
I'm not even sure what "embargo" is, but interested as to why these changes made caching change.

Reply:
This is because now the current role-based validations and the new permission-based validations coexist and run together, so the number of database queries increases, as the role-based validations are deprecated this number will decrease again